StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1642UnitTests.cs

```diff
@@ -20,9 +23,28 @@ namespace StyleCop.Analyzers.Test.DocumentationRules
     [UseCulture("en-US")]
     public class SA1642UnitTests
     {
+        public static IEnumerable<object[]> AllTypeKinds
```

Comment:
This is already defined here:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/890236e1bef6f943554cfdc3b13e2cf2c8651863/StyleCop.Analyzers/StyleCop.Analyzers.Test/Helpers/CommonMemberData.cs#L14-L32

You can use it like this:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/890236e1bef6f943554cfdc3b13e2cf2c8651863/StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1502/SA1502UnitTests.Methods.cs#L28-L29

Reply:
Missed it 👍
Updated.